apps/web/app/(use-page-wrapper)/settings/(settings-layout)/developer/webhooks/(with-loader)/loading.tsx

```diff
@@ -0,0 +1,14 @@
+import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
+import { SkeletonLoader } from "@calcom/features/webhooks/pages/webhooks-view";
+
+export default function Loading() {
```

Comment:
looks like this while loading

<img width="1512" alt="Screenshot 2025-04-07 at 5 53 56 AM" src="https://github.com/user-attachments/assets/10e73953-d70c-4d66-bd5c-f2863bf7427c" />